src/wavesurfer.ts

```diff
@@ -140,6 +141,9 @@ class WaveSurfer extends Player<WaveSurferEvents> {
   protected subscriptions: Array<() => void> = []
   protected mediaSubscriptions: Array<() => void> = []
 
+  public static readonly BasePlugin = BasePlugin
+  public static readonly createElement = createElement
```

Comment:
Sorry for the back and forth on this, but now that it's a static method of WaveSurfer, it would be good to not name it similar to the existing `create` method to avoid confusion when using autocomplemte.

I suggest we do

```js
import * as dom from './dom.js'

// ...

  public static readonly dom = dom
```

`createElement` could then be a named export.

Reply:
No worries at all dude! I updated the code.

`*` only imports named exports from a file. So I needed to update `dom.js` to have named export `createElement`.


I think we'd need to leave the default export since others maybe importing the default from `dom.js` in their projects.